pennylane/math/single_dispatch.py

```diff
@@ -138,6 +152,29 @@ def _take_autograd(tensor, indices, axis=None):
 )
 
 
+def _ifft2_tf(a, s=None, axes=(-2, -1), norm=None):
+    if axes != (-2, -1):
+        raise ValueError(
+            "TensorFlow does not support passing axes; the ifft "
+            "will always be performed over the inner-most 2 dimensions."
+        )

```

Comment:
Would it be good to check also if `s` is something other than `None`? since it is not used?

Reply:
Yep! Definitely.